pkgs/development/tools/gdm/default.nix

```diff
@@ -0,0 +1,25 @@
+# This file was generated by go2nix.
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "gdm-${version}";
+  version = "1.4";
+
+  goPackagePath = "github.com/sparrc/gdm";
+
+  src = fetchgit {
+    url = "https://github.com/sparrc/gdm";
+    sha256 = "0kpqmbg144qcvd8k88j9yx9lrld85ray2viw161xajafk16plvld";
+    rev = version;
```

Comment:
oh, usually I replace fetchgit manually. However in this case I forgot it. @kamilchm if you take a look at how `nix-prefetch-zip` is implemented, it should be also possible to generate `fetchFromGitHub` from downloaded source (the hash is calculated using the content of the directory, which should be the same as the repo). This should you also give a nice speed-up because fetchgit is kind of slow for both downloading and disk i/o.